src/radio/index.scss

```diff
@@ -2,10 +2,22 @@
   $size: 22px;
   $padding: 8px;
 
+  margin: 8px 0 0 0;
+  padding: 0;
+  border: none;
+
+  > .ui-label {
+    margin-bottom: -3px;
+  }
+
   .c-radio-item {
     margin-top: 8px;
     display: block;
 
+    &:first-child {
```

Comment:
Fieldset collapses margin in chrome, but not in Firefox. That's why I have to manually unset top margin if there is no label.